### ISSUE TITLE
[1.16.4] Crash on Startup

### ISSUE THREAD

**Issue by Portalmaker233** [MC-U-Team/Music-Player#128]
**Describe the bug**
I failed to load this amazing mod,can you figure out what the problem is?Everytime I try to load this mod,it says:Invalid escape sequence at line 179 column 37 path $.playlists[5].uris[0].o

**Versions**
- Mod version: 2.2.1.116
- Minecraft version: 1.16.4
- Forge version: 35.1.37
- UTeamCore version: 3.1.17.191

**Log (if crash occures)**
Link to crash log
[crash-2021-01-18_11.34.30-fml.txt](https://github.com/MC-U-Team/Music-Player/files/5827672/crash-2021-01-18_11.34.30-fml.txt)

**Comment by MrTroble** [comment]
We will take a look at this issue. For now you can delete the `playlist.json.gz` in your config file. Default it can be found under `config/musicplayer` or `%appdata%\.minecraft\mc-u-team\musicplayer`

**Comment by MrTroble** [comment]
Or you can send it in and I can try to repair it.

**Comment by MrTroble** [comment]
@HyCraftHD Maybe you just need a additional check in your catch phrase to catch the MalformedJsonException

**Comment by HyCraftHD** [comment]
The bug seems to be caused by chinese characters, which must be fixed on some other end.
@Portalmaker233 Can you upload the playlist.json.gz file here on github?

**Comment by Portalmaker233** [comment]
The things are getting serious,I can't able to find the file you guys let me give.

**Comment by Portalmaker233** [comment]
The things are getting serious suddenly,I'm unable to find the file you guys just asked for.

**Comment by Portalmaker233** [comment]
Will this be helpful?
[2021-01-19-1.log.gz](https://github.com/MC-U-Team/Music-Player/files/5833992/2021-01-19-1.log.gz)

**Comment by HyCraftHD** [comment]
Not really. Can you go to the config folder and set internalPlaylists to true in the musicplayer config file.
After that can you try to repeat what you did before it break?
If it breaks please look in youminecraftinstallation/config/musicplayer and send me the playlist.json.gz

**Comment by Portalmaker233** [comment]
I set internalPlaylists to true and still fail to start my game,and I did nothing to this mod,I just uninstall this mod and change the version again and again before I asked you guys for help,as a result,this is useless.

**Comment by Portalmaker233** [comment]
And what confused me most is that I can't find the file called playlist.json.gz,I used my searching function on my windows 10 and for nothing.

**Comment by Portalmaker233** [comment]
Wait!I found this file！
[playlist.json.gz](https://github.com/MC-U-Team/Music-Player/files/5938650/playlist.json.gz)

**Comment by HyCraftHD** [comment]
Thanks I'll have a look at it soon
